src/lib/components/addons/History.svelte

```diff
@@ -50,6 +55,10 @@
 
   {#if loading}
     <Empty icon={Hourglass24}>Loading past runs…</Empty>
+  {:else if error}
+    <Empty icon={Alert16}>
```

Comment:
Larger icon variants work better in this slot:

```suggestion
    <Empty icon={Alert24}>
```